doc/getting-started/overview_dev.rst

```diff
@@ -116,29 +116,27 @@ meet their requirements without pre-launched VMs. Even if your application has
 stringent real-time requirements, start by testing the application on a
 post-launched VM before considering a pre-launched VM.
 
-Predefined Scenarios
----------------------
+Scenario Types
+---------------
 
-To help accelerate the configuration process, ACRN offers the following
-:ref:`predefined sample scenarios <usage-scenarios>`:
+ACRN categorizes scenarios into :ref:`three types <usage-scenarios>`:
 
 * **Shared scenario:** This scenario represents a traditional computing, memory,
   and device resource sharing model among VMs. It has post-launched User VMs and
   the required Service VM. There are no pre-launched VMs in this scenario.
 
-* **Partitioned scenario:** This scenario has pre-launched User VMs to
-  demonstrate VM partitioning: the User VMs are independent and isolated, and
+* **Partitioned scenario:** This scenario has pre-launched User VMs only. It
+  demonstrates VM partitioning: the User VMs are independent and isolated, and
   they do not share resources. There is no need for the Service VM or Device
```

Comment:
It's worth adding that storage devices are not shared either, so each pre-launched VM requires its own boot device.